pkgs/tools/misc/aescrypt/default.nix

```diff
@@ -9,6 +9,12 @@ stdenv.mkDerivation rec {
     sha256 = "1a1rs7xmbxh355qg3v02rln3gshvy3j6wkx4g9ir72l22mp6zkc7";
   };
 
+  makeFlags = [ "CC=cc" ];
+
+  patches = [ ./support-darwin.patch ];
+
+  NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isDarwin "-liconv";
```

Comment:
@matthewbauer Any idea why just adding `libiconv` isn't enough, shouldn't that just work now?